quantum/keymap_extras/keymap_swedish_osx_ansi.h

```diff
@@ -0,0 +1,233 @@
+/* Copyright 2021
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "keymap.h"
+
+// clang-format off
+
+/*
+ * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬─────┐
+ * │ < │ 1 │ 2 │ 3 │ 4 │ 5 │ 6 │ 7 │ 8 │ 9 │ 0 │ + │ ´ │     │
```

Comment:
Sublime Text 4 now renders NBSPs... differently:
![image](https://user-images.githubusercontent.com/4781841/122659368-db66c600-d1ba-11eb-8f11-6360cc005474.png)
So my clever technique to make these readable even with whitespace shown, is not so clever anymore :(

If you could change them all to spaces? (I'll fix the rest of the headers at some point later...)

Reply:
👍 

I have updated the files and replaced the NBSPs with spaces.